config.cpu.yaml

```diff
@@ -34,11 +39,11 @@ embedding:
     n_batch: 512
     n_ctx: 8192
 
-  hugging_face:
+  #hugging_face:
     # model_name: all-MiniLM-L6-v2
-    model_name: sentence-transformers/all-mpnet-base-v2
-    model_kwargs:
-      device: cpu
+    #model_name: sentence-transformers/all-mpnet-base-v2
+    #model_kwargs:
+    #  device: cpu
```

Comment:
let's leave it uncommented, same for the gpu config

Reply:
Can't because the new model is hugging_face as well

Reply:
ah sorry didn't see it was inside embedding section. We can get rid of it then since it is the same thing as the top one.